docs/developer-docs/integrations/bitcoin/ckbtc.md

```diff
@@ -0,0 +1,37 @@
+# Chain-key Bitcoin
+
+Chain-key Bitcoin (ckBTC) is an [ICRC-1](https://github.com/dfinity/ICRC-1/blob/aa82e52aaa74cc7c5f6a141e30b708bf42ede1e3/standards/ICRC-1/README.md) compliant token that
+is backed 1:1 by bitcoin held 100% on-chain.
+
+The ckBTC functionality is provided through an interplay of two canisters,
+the _ckBTC minter_ and the _ckBTC ledger_.
+
+The ckBTC minter is responsible for the minting and burning of ckBTC tokens.
```

Comment:
We would like to avoid the terms "mint" and "burn" for ckBTC. Is that feasible?

Reply:
I would say it's not feasible. The canister is even called ckBTC **minter**.

It doesn't make a lot of sense to try to avoid these terms because they are used a lot in the [ICRC-1 standard](https://github.com/dfinity/ICRC-1/blob/main/standards/ICRC-1/README.md). Since ckBTC adheres to this standard, it obviously supports minting and burning.

Reply:
OK, reasonable, in this case let's continue using the terms here.